Rmarkdown/Multiple_Ascending_Dose_PKPD_continuous.Rmd

```diff
@@ -220,61 +224,76 @@ gg <- gg + guides(color = guide_legend(""), fill = guide_legend(""))
 gg <- gg + xgx_scale_y_log10() 
 gg <- gg + labs(y = conc_label)
-print(gg)
+
+gg1 <- gg + geom_point(data = pk_data %>% subset(CENS==1,), color="red", shape=8, alpha = 0.5)
+print(gg1)
 
 #PD data
 gg %+% (data = pd_data %>% subset(DOSE>0)) + scale_y_continuous() + labs(y = pd_label)
 ```
 
 ## Explore Exposure-Response Relationship
+
+### PD marker by Concentration, for endpoint of interest, mean (95% CI) by concentration bins
 <!--START_EXPLANATION-->
 Plot PD marker against concentration. Do you see any relationship? Does response increase (decrease) with increasing dose? Are you able to detect a plateau or emax (emin) on the effect? 
 
 **Warning:** Even if you don't see an Emax, that doesn't mean there isn't one. Be very careful about using linear models for Dose-Response or Exposure-Response relationships. Extrapolation outside of the observed dose range could indicate a higher dose is always better (even if it isn't).
 <!--END_EXPLANATION-->
+
 ```{r, error = TRUE, cache = TRUE, fig.width = 8, fig.height = 4, warning = FALSE, message= FALSE}
 gg <- ggplot(data = pkpd_data_wide %>% subset(PROFDAY == SS_PROFDAY,), aes(x = CONC,y = PD))
 gg <- gg + xgx_annotate_status(status)
 gg <- gg + geom_point(aes(color = TRTACT_high2low))
 gg <- gg + geom_point(data = pkpd_data_wide %>% subset(CENS==1,), color="red", shape = 8)
 gg <- gg + labs(x = conc_label , y = pd_label)
-gg <- gg + xgx_scale_x_log10() 
 gg <- gg + guides(color = guide_legend(""))
-print(gg)
+gg <- gg + xgx_geom_smooth_emax(color = "black") 
+gg <- gg + xgx_stat_ci(bins = 4, geom = "point", shape = 0, size = 4)
+gg <- gg + xgx_stat_ci(bins = 4, geom = "errorbar")
+
+gg1 <- gg + ggtitle("Concentration on Linear Scale")
+gg2 <- gg + xgx_scale_x_log10() + ggtitle("Concentration on Log Scale")
+
+grid.arrange(gg1,gg2,nrow = 1)
 ```
 
-```{r, error = TRUE, cache = TRUE, fig.width = 8, fig.height = 3, warning = FALSE, message= FALSE}
+### PD marker by Concentration, faceted by visit, mean (95% CI) by concentration bins
+
+```{r, error = TRUE, cache = TRUE, fig.width = 8, fig.height = 6, warning = FALSE, message= FALSE}
 data_to_plot <- pkpd_data_wide %>% subset(PROFDAY %in% PD_PROFDAYS,)
 
 gg <- ggplot(data = data_to_plot,
-             aes(x = CONC, y = PD, color = TRTACT_high2low))
+             aes(x = CONC, y = PD))
 gg <- gg + xgx_annotate_status(status)
-gg <- gg + geom_point()
+gg <- gg + geom_point(aes(color = TRTACT_high2low))
 gg <- gg + geom_point(data = data_to_plot %>% subset(CENS==1,), color="red", shape = 8)
 gg <- gg + guides(color = guide_legend(""), fill = guide_legend(""))
 gg <- gg + labs(x = conc_label , y = pd_label)
-gg <- gg + xgx_scale_x_log10() 
-gg + facet_grid(~DAY_label)
+gg <- gg + xgx_stat_ci(bins = 4, geom = "point", shape = 0, size = 4)
+gg <- gg + xgx_stat_ci(bins = 4, geom = "errorbar")
+gg <- gg + facet_grid(~DAY_label)
+
+gg1 <- gg + xgx_geom_smooth_emax(color = "black") + ggtitle("Concentration on Linear Scale")
```

Comment:
I wonder if every time you call xgx_stat_smooth_emax() if it would be preferable to show how to do it when specifying the initial conditions. In my experience, I often have to do that.  But if it's not shown how to do so in the example, then if people get an error, they might struggle a bit figuring out how to change the ICs.